build.gradle

```diff
@@ -119,6 +119,9 @@ allprojects {
   repositories {
     mavenCentral()
     mavenLocal()
+    maven {
+      url "https://repository.apache.org/content/repositories/snapshots/"
```

Comment:
Need this for now because this PR uses `4.0.1-SNAPSHOT`. 
Will remove this in the future